### DISCUSSION TITLE
How do I change the output language

### DISCUSSION THREAD

**Discussion by AssistMelody** [tldr-pages/tldr#10536] [Q&A] [answered]
Sorry I didn't find a document for the change lang

**Comment by kbdharun** [comment, marked as answer]
Hi, [Most of our clients automatically detect your locale from your system configuration](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#language) and display the translated pages for the commands (if it exists, or else it fallbacks to English). While, some of our clients like our [Python client](https://github.com/tldr-pages/tldr-python-client#configuration) allow you to set the locale in your shell's configuration file (i.e `.bashrc`, etc). If the client doesn't read any configuration file, it would probably have an option like `-l` or `--language` to specify or override the locale.

I would suggest checking your client's documentation regarding their implementation. In case, your client (maybe old ones) doesn't have language support, I would suggest [switching to a client that already has it](https://github.com/tldr-pages/tldr/wiki/tldr-pages-clients).

**Reply by AssistMelody** [reply]
thks i get it
